executor/linux/build.go

```diff
@@ -370,6 +370,7 @@ func (c *client) AssembleBuild(ctx context.Context) error {
 
 		// assume no privileged images are in use
 		containsPrivilegedImages := false
+		privImage := ""
```

Comment:
thoughts on making this a slice, so in the scenario multiple images get denied we can log them all with `%v`?

Reply:
oh yeah that's a good idea! done!